### PR TITLE
Switchboard check-in PR

### PR DESCRIPTION
**Pull request by max-cura**

### CODE REVIEW

**Comment by lschuermann** [comment]
I don't know why the end-to-end tests here are failing, but presumably that would be something that we'd want to fix in a follow-up PR. It might be a fluke, given that this version of the switchboard currently works with the QEMU supervisor on the `main` branch.

**Comment by lschuermann** [comment]
@charles37 I'm on it, @max-cura sent me an example of the new API semantics yesterday from which I'm reverse engineering this.

**Comment by charles37** [comment]
> @charles37 I'm on it, @max-cura sent me an example of the new API semantics yesterday from which I'm reverse engineering this.

Can you send this to me as well so I can synchronously change the routes on the CLI tool branch?

**Comment by lschuermann** [comment]
This is the httpie invocation: `noglob https --print='HBhbm' -A bearer -a "${tml_token}" POST api.treadmill.ci/api/v1/job/queue request_id="$(uuidgen)" init_spec[type]=image init_spec[image_id]="e4673d891ecb0f40f89ec2a92677eeac6d06b746722df7e5d2b674068a7aa7f1" ssh_keys:='["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC9XgOJNVpO00+p04GwoL0viFco4p24PrsPzvsCOiAETJ6ttYiOFjYrpjQvNLsDse0PcW+duhtjyyp+Y7JZaOTeG6FX1Y8j6gDCUML58f4NlKtnWzfMftYkVO8QeYzdjJhG3J2zBXuqMmen7elVgZouvM/H47X620q7BssTTS1YIVnXxNn5jip8UbSJ1073MnUuTjSGrmS9yyLQx4Ka9/u6zzPDASw6OWXyzoXkWVpU5VzAqYk8Ob9C+lFQw5LMERsQBADoNB+kg/m+OoKS7XXu9+WFdTKsqhy7/c6hijRPaNsP/JRfsxEyjF+Y8LyokU2OXq0MWM0xZrt8O3/DsXAKqyIrGCVPQX+eeXvqP8LusFm2CDe8zoUeyLXvBdZX6Xxyy00OHHQRsuIbBYCJOUMbmDwIcEaC4DELjcNFZXJQYhj2hvB2sQvTMjnS58FDkD+IgVTTlTTzkEiDwbIV6lHbSqzA6AvZtZXd1+rQ4wFIT0clQhAGpBWKN+8Cotl51yC+P+nbU9xlCXM17Q3Pev2sjZFx1VU3oa5SEzEv74aWhUDCaATun/ebi/1Scm3ekOsiPVHuUBm9a2GUMYIuAOTL6W9AM2zZJ8xWkJw5Rj1eQnIDwD9izCRR1gJXlkTYwLn6ZnLawX6FS6Vvj5NV8mjjU4xlhnwaWF0b/Jmm3E966w=="]' restart_policy[remaining_restart_count]:=0 ssh_rendezvous_servers:='[]' parameters:='{}' tag_config='supervisor:7d55ec6d-15e7-4b84-8c04-7c085fe60df4' override_timeout:=null;`

(Using the example token from our fixtures, so fine to publish here)

File: connector/ws/src/lib.rs [hunk 1]
```diff
@@ -61,6 +59,7 @@ struct Inner<S: connector::Supervisor> {
 
 impl<S: connector::Supervisor> WsConnector<S> {
     pub fn new(supervisor_id: Uuid, config: WsConnectorConfig, supervisor: Weak<S>) -> Self {
+        tracing::error!("UH OH");
```
Comment (lschuermann) [inline connector/ws/src/lib.rs:62]: Hm? :laughing:

File: switchboard/sql/SCHEMA.sql [hunk 2]
```diff
@@ -133,8 +133,8 @@ create type job_known_state as enum (
     -- The job is currently running. In a precise sense, the real meaning is "the job MAY be running", however, the
     -- switchboard should treat "may be running" as "is running"; thus, this state is called 'running'.
     'running',
-    -- The job has run to completion.
-    'finished');
+    -- The job is not running and does not need to run.
+    'not_queued');
 create table jobs
```
Comment (lschuermann) [inline switchboard/sql/SCHEMA.sql:138]: What would we need this state for exactly?

**Review by lschuermann — DISMISSED** [review]

File: connector/ws/src/lib.rs (hunk 1 above)
Comment (max-cura) [reply connector/ws/src/lib.rs:62]: Ah, extra debugging statement that I forgot to remove.

File: switchboard/sql/SCHEMA.sql (hunk 2 above)
Comment (max-cura) [reply switchboard/sql/SCHEMA.sql:138]: The `known_state` type is intended to be used when the switchboard is reconstructing state after having to restart.

**Pull request merged**